### PR TITLE
Feature/location show one

### PR DESCRIPTION
**Pull request by lcsdtw**
:warning: _This fixes make run for master_

### Requirement 
<details>
    <summary>
        <strong>Fix make run which has a conflict of LocationView</strong> 
    </summary>
    <p>
       There is a class LocationView at airports and locations now.
    </p>
</details>

### Solution
<details>
    <summary>
        <strong>Rename class LocationView from Airports</strong>
    </summary>
    <p>
        Given theses classes are started by beans, they conflict when trying to start project. Renaming the one at Airports solves that.
    </p>
</details>

### CODE REVIEW

File: src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 import br.com.caelum.clines.api.airports.ExistingLocationService;
 import br.com.caelum.clines.shared.domain.Location;
+import br.com.caelum.clines.shared.infra.Mapper;
 import org.springframework.data.repository.Repository;
 
 public interface LocationRepository extends Repository<Location, Long>, ExistingLocationService {
+    void save(Location location);
```
Comment (MarceloLeite2604) [inline src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java]: Spring interface `Repository` already has a method `save(E entity)`. I don't think it is necessary redeclarate it.

**Review by MarceloLeite2604 — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ services:
     environment:
       POSTGRES_PASSWORD: postgres
       POSTGRES_DB: clines
+    ports:
```
Comment (fwfurtado) [inline docker-compose.yml]: do we need to expose that door?

File: src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java (hunk 1 above)
Comment (fwfurtado) [reply src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java]: `save(E entity)` is only available in `CrudRepository` or `PaggingAndSortRepository`

File: src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java [hunk 3]
```diff
@@ -1,5 +1,9 @@
 package br.com.caelum.clines.shared.exceptions;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```
Comment (fwfurtado) [inline src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java]: Given that we have a `GlobalExceptionHandler`, it makes sense keep this annotation?

**Review by fwfurtado — CHANGES_REQUESTED** [review]

**Review by marialuisacp — APPROVED** [review]

File: src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java (hunk 1 above)
Comment (MarceloLeite2604) [reply src/main/java/br/com/caelum/clines/api/locations/LocationRepository.java]: My mistake then.

**Review by MarceloLeite2604 — APPROVED** [review]

File: src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java (hunk 3 above)
Comment (lcsdtw) [reply src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java]: The test does not work without it

Comment (lcsdtw) [reply src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java]: (We will review this code, yesterday we hadn't the example code)

File: docker-compose.yml (hunk 2 above)
Comment (lcsdtw) [reply docker-compose.yml]: Removed

File: src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java (hunk 3 above)
Comment (lcsdtw) [reply src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java]: The test still does not works without it

Comment (lcsdtw) [reply src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java]: Wrong exception type. Fixed now.

**Review by fwfurtado — APPROVED** [review]

**Pull request merged**